### PR TITLE
Exclude parameter

### PR DESCRIPTION
**Pull request by billzabaronick**
This is the code to pass an exclude parameter to exclude a list of rules; addresses issue #127 

ex: apigeelint -s apiproxy -f table.js -e PO006,PO008

### CODE REVIEW

File: lib/package/bundleLinter.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ var getFormatter = function(format) {
 var executePlugin = function(file, bundle, callback) {
   if (file.endsWith(".js")) {
     var plugin = require("./plugins/" + file);
-    if (plugin.plugin.enabled) {
+    if (plugin.plugin.enabled && !excluded.included(plugin.ruleId)) {
```
Comment (apetrosian) [inline lib/package/bundleLinter.js]:
```suggestion
    if (plugin.plugin.enabled && !excluded.includes(plugin.plugin.ruleId) ) {
```

File: lib/package/bundleLinter.js [hunk 2]
```diff
@@ -179,7 +179,7 @@ var getFormatter = function(format) {
 var executePlugin = function(file, bundle, callback) {
```
Comment (apetrosian) [inline lib/package/bundleLinter.js]: `excluded` array is not defined here, will produce an error

File: cli.js [hunk 3]
```diff
@@ -37,6 +38,10 @@ if (program.formatter) {
   configuration.formatter = program.formatter || "json.js";
 }
 
+if (program.excluded && typeof(program.excluded) === "string") {
+	configuration.excluded = program.excluded.split(",");
```
Comment (apetrosian) [inline cli.js]: I would recommend to add a default value, e. g. empty array to avoid additional checks in other places

Comment (billzabaronick) [reply cli.js]: I added a default value in the initialization of configuration

File: lib/package/bundleLinter.js (hunk 2 above)
Comment (billzabaronick) [reply lib/package/bundleLinter.js]: this was missing. I had the wrong code in the original pull request.

File: lib/package/bundleLinter.js (hunk 1 above)
Comment (billzabaronick) [reply lib/package/bundleLinter.js]: I updated this check to better reflect the data structure.

**Pull request merged**